bin/hit/_hit

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env bash
 
 HIT_SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-$_}")")"
-HIT_PROJECT="$(basename "$(cd "$HIT_SCRIPT_DIR/../../" && pwd)")"
 HIT_WAGON_NAMES=$(ls -d "$HIT_SCRIPT_DIR"/../../app/hitobito_* | sed 's|.*/hitobito_||')
 export HIT_SCRIPT_DIR
-export HIT_PROJECT
 export HIT_WAGON_NAMES
 source "$HIT_SCRIPT_DIR/welcome_info"
 
-PS1="HIT-$HIT_PROJECT> "
```

Comment:
same reason as above ... it's good to know in which project I am.

as a developer I often have to work with multiple wagon setups